src/Core/tests/Benchmarks/Program.cs

```diff
@@ -6,8 +6,7 @@ class Program
 	{
 		static void Main(string[] args)
 		{
-			//BenchmarkRunner.Run<MauiServiceProviderBenchmarker>();
-			BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).RunAllJoined();
+			BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
```

Comment:
This gives a better experience when you run this project:
```
> dotnet run --project C:\src\maui\src\Core\tests\Benchmarks\Core.Benchmarks.csproj -c Release
Available Benchmarks:
  #0 GetHandlersBenchmarker
  #1 ListUnionBenchmarker
  #2 MauiServiceProviderBenchmarker
  #3 RegisterHandlersBenchmarker
  #4 ShellBenchmarker


You should select the target benchmark(s). Please, print a number of a benchmark (e.g. `0`) or a contained benchmark caption (e.g. `GetHandlersBenchmarker`).
If you want to select few, please separate them with space ` ` (e.g. `1 2 3`).
You can also provide the class name in console arguments by using --filter. (e.g. `--filter *GetHandlersBenchmarker*`).
```

Probably no one would want to run them all.